conduct.rst

```diff
@@ -0,0 +1,252 @@
+a steady stream of them compounds
+to make people in under-represented groups feel less welcome.
+
+Time
+====
+
+Be respectful of other people:
+their time, their location,
+and other considerations of a distributed team.
+
+Be on time to meetings.
+Read the meeting agenda ahead of time,
+and if something requires your input
+and you cannot make the meeting,
+tell the meeting organizer.
+
+Resist the temptation
+to have local-only meetings
+when remote teammates should be involved—take them to the forum,
+a group hangout, Github issue, etc. as appropriate.
+
+Keep in mind that what you write once
+will be read by several people.
+Writing a short message means people
+can understand the conversation as efficiently as possible.
+When a long explanation is necessary,
+consider adding a summary.
+Try to bring new arguments to a conversation
+so that each message adds something unique to the thread,
+keeping in mind that the rest of the thread
+sill contains the other messages that have already been made.
+
+Try to stay on topic,
+especially in discussions that are already fairly large.
+If a discussion brings to mind some related
+but separate issue,
+please start a separate discussion for it.
+Everyone that is participating in an existing discussion
+will naturally wish to respond to your comment.
+If it is a separate discussion,
+each person can choose to join that discussion
+or spend their time some other way.
+
+Giving and receiving feedback
+=============================
+
+Give constructive, not critical feedback.
+Feedback is negatively critical
+when it surfaces something wrong with someone or something they produced,
+especially without any mention
+of ways to make their behavior or their product better.
+Critical feedback on work often looks like
+"you don't write enough tests"
+or "your code quality isn't good enough."
+Personal criticism can be more severe
+and often looks like "you should be less judgmental"
+or "you are a burden because you ask too many questions."
+Constructive feedback
+is more about how a person can do better
+rather than what they are doing wrong.
+If you want someone to do something better,
+you should tell them what better looks like.
+Ask a question to get a discussion rolling,
+to gain context,
+and then if you see room for improvement
+give declarative feedback to that effect.
+This creates an environment
+where people understand what success looks like
+instead of just feeling like they are unsuccessful.
+
+Code, configurations, and their reviews
+are also mechanisms for communication.
+Just as you shouldn't interact with people poorly in person,
+do not interact poorly through code or code review.
+
+You are not your products.
+Technical critiques are integral,
+and should be hard on the product,
+not on the producer.
+While it is important to care about your work
+and to produce the best thing you can,
+this can make review difficult.
+It is important to realize that it's better
```

Comment:
I'd recommend adding something like this here (before the 'it is important' bit):

As mentioned, the onus is on the writer of a comment to be constructive in their feedback.  Similarly, there is an onus on the reader to read a comment as attempting to be constructive and not personally critical -- as hard as this may be when you are passionate about what you do.